apps/web/src/components/error/error-demo.tsx

```
@@ -0,0 +1,136 @@
+        throw new Error(`Render error triggered in ${level} level error boundary test`);
+      
+      case 'async':
+        // Async errors won't be caught by error boundaries, but we can simulate
+        setTimeout(() => {
+          throw new Error(`Async error triggered in ${level} level - this won't be caught by error boundary`);
+        }, 100);
+        break;
+        
+      case 'null-reference':
+        // Null reference error
+        const nullObj: any = null;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)